app/src/main/java/org/mozilla/fenix/tabtray/TabTrayDialogFragment.kt

```diff
@@ -71,7 +71,11 @@ class TabTrayDialogFragment : AppCompatDialogFragment(), UserInteractionHandler
     private lateinit var tabTrayDialogStore: TabTrayDialogFragmentStore
 
     private val snackbarAnchor: View?
-        get() = if (tabTrayView.fabView.new_tab_button.isVisible) tabTrayView.fabView.new_tab_button
+        get() = if (tabTrayView.fabView.new_tab_button.isVisible || tabTrayView.mode != Mode.Normal) tabTrayView.fabView.new_tab_button
```

Comment:
This line is too long and detekt throws a `MaxLineLength` error. 
I recommend running `./gradlew detekt` and ` ./gradlew ktlint` before pushing.
Otherwise it looks and works great! Thanks for taking a look at this. 🥇

Reply:
Ohh, I didn't know about that! I'm fixing it right now @codrut-topliceanu .💨